pkgs/applications/editors/vim/plugins/vim-plugin-names

```diff
@@ -643,6 +643,7 @@ https://github.com/folke/tokyonight.nvim/,,
 https://github.com/markonm/traces.vim/,,
 https://github.com/tjdevries/train.nvim/,,
 https://github.com/tremor-rs/tremor-vim/,,
+https://github.com/cappyzawa/trim.nvim/,HEAD,
```

Comment:
Why do we need head here?

Reply:
Hm, not sure, that's what the script added by itself, let me remove it.